docker/docker-compose.dev.yml

```diff
@@ -19,6 +19,8 @@ services:
     depends_on:
       - redis
       - postgres
+    stdin_open: true
+    tty: true
```

Comment:
I think this is generally useful for debugging.

Reply:
ok